gobblin-utility/src/main/java/gobblin/util/SchedulerUtils.java

```diff
@@ -178,6 +210,81 @@ public boolean accept(File file) {
   }
 
   /**
+   * Recursively load job configuration files under given URI of **directory of config files folder**
+   */
+  private static void loadGenericJobConfigsRecursive(List<Properties> jobConfigs, Properties rootProps,
+      Set<String> jobConfigFileExtensions, Path configDirPath)
+      throws ConfigurationException, IOException {
+
+    /* create the generic file system */
+    Configuration conf = new Configuration();
+    try (FileSystem filesystem = configDirPath.getFileSystem(conf)) {
+      if (!filesystem.exists(configDirPath)) {
+        throw new RuntimeException("No specified .properties directory found :" + configDirPath.toString());
```

Comment:
Can you change the log message to something like `"The specified job configurations directory was not found: "`